nservicebus/hosting/ibus-instance.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Managing IBus instance
+summary: Fluent configuration API to get transnational one-way messaging
+tags:
+- Hosting
+- IOC
+- Dependency Injection
+- Service Locator
+---
+
+
+When you have created a `IBus` instances then you must register it somewhere so that your application can use it to send messages.
+
+There are various ways in managing and storing the `IBus` instance.
+
+* Global static
+* Using dependency injection
+
+The first option creates tight coupling the last loose coupling.
+
+
+
+## Global static
+
+After you have created the bus you assign the object to a global static variable so that the bus can be used in other locations
+
+
+Initializing:
+
+```
+public class Global
```

Comment:
this is kind of out of context. clearly this is copied from a web application. but anywhere else why would you name something as generic as global.